components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/java/org/wso2/carbon/apimgt/internal/service/impl/GatewaySynapseArtifactsApiServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package org.wso2.carbon.apimgt.internal.service.impl;
+
+import org.wso2.carbon.apimgt.api.APIManagementException;
+import org.wso2.carbon.apimgt.impl.dao.GatewayArtifactsMgtDAO;
+import org.wso2.carbon.apimgt.internal.service.GatewaySynapseArtifactsApiService;
+import org.apache.cxf.jaxrs.ext.MessageContext;
+import org.wso2.carbon.apimgt.internal.service.dto.SynapseArtifactListDTO;
+import java.util.List;
+import javax.ws.rs.core.Response;
+
+
+
```

Comment:
remove additional new line

Reply:
Fixed in the latest version https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/java/org/wso2/carbon/apimgt/internal/service/impl/GatewaySynapseArtifactsApiServiceImpl.java#L12